src/test/java/uk/gov/hmcts/reform/sendletter/tasks/SerialTaskRunnerLockingTest.java

```diff
@@ -0,0 +1,153 @@
+package uk.gov.hmcts.reform.sendletter.tasks;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import javax.sql.DataSource;
+
+import static org.mockito.Matchers.startsWith;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class SerialTaskRunnerLockingTest {
+
+    @Mock
+    private DataSource source;
+    @Mock
+    private Connection connection;
+    @Mock
+    private Statement statement;
+    private SerialTaskRunner taskRunner;
+
+    @Before
+    public void setup() throws SQLException {
+        when(connection.createStatement()).thenReturn(statement);
+        when(source.getConnection()).thenReturn(connection);
+
+        taskRunner = SerialTaskRunner.get(source);
+    }
+
+    @Test
+    public void successfully_locks_and_unlocks() throws SQLException {
+        lockedWithSuccess();
+        unlockedWithSuccess();
```

Comment:
looks like this is mocking responses for future operations on `source`...
perhaps `locking/unlocking will succeed/fail`?